PURL-TYPES.rst

```diff
@@ -461,9 +461,11 @@ pypi
 - PyPI treats ``-`` and ``_`` as the same character and is not case sensitive.
   Therefore a PyPI package ``name`` must be lowercased and underscore ``_``
   replaced with a dash ``-``.
+- The ``filename`` qualifier selects a particular distribution file (case-sensitive).
```

Comment:
Using the name instead of individual tags is much better, thank you! Because naming is hard, I figured I'd throw `dist` out there as an option ;) Either way I'm happy with this.

Reply:
IMO `filename` is better, `dist` feels a bit like jargon.

Reply:
Agreed. In fact, I struggled for what to call it in the text. Actually, should I say "distribution package file" to be unambiguous? (I want to clarify that it's not a file *within* an archive, which would instead be the `#path` part of the URL.)

Reply:
I agree too, let's go with `filename` :shipit: